disnake/widget.py

```diff
@@ -132,21 +133,21 @@ class WidgetMember(BaseUser):
     Attributes
     ----------
     id: :class:`int`
-        The member's ID.
+        The member's anonymized ID.
     name: :class:`str`
         The member's username.
     discriminator: :class:`str`
-        The member's discriminator.
-    bot: :class:`bool`
-        Whether the member is a bot.
+        The member's anonymized discriminator.
     status: :class:`Status`
         The member's status.
     nick: Optional[:class:`str`]
         The member's nickname.
-    avatar: Optional[:class:`str`]
-        The member's avatar hash.
+    avatar: Optional[:class:`Asset`]
+        The user's avatar. The size can be chosen using :func:`Asset.with_size`,
+        however the format is always static and cannot be changed
+        through :func:`Asset.with_format` or similar methods.
```

Comment:
unless the documentation was wrong and this was an asset before, this should have a version changed note.

Reply:
Documentation was wrong, this was previously just `BaseUser.avatar`, which would coincidentally always return `None` in `WidgetMember`s.